lib/cisco_node_utils/cmd_ref/itd_device_group.yaml

```diff
@@ -0,0 +1,83 @@
+# itd_device_group
+---
+_exclude: [N3k, N5k, N6k, ios_xr]
```

Comment:
We need to verify if this is supported on N8k.  I will send you an email with connection details to my N8k to verify the cli.